src/app/guards/index.ts

```diff
@@ -3,3 +3,5 @@ export * from './admin.guard';
 export * from './auth-required.guard';
 export * from './roles.gurad';
 export * from './data-view-role.guard'
```

Comment:
When we roles guard, why do we need another as data view guard.

For now its okay to keep to avoid last minute issues, but add a Jira to simplify role based access guards.